src/logger.js

```diff
@@ -1,143 +1,118 @@
-import _ from 'lodash';
-import semver from 'semver';
-import net from 'net';
-import tls from 'tls';
-import urlUtil from 'url';
-import { Writable } from 'stream';
-import codependency from 'codependency';
-import reconnectCore from 'reconnect-core';
-import * as defaults from './defaults';
-import * as levelUtil from './levels';
-import text from './text';
-import build from './serialize';
-import {
-    BadOptionsError,
-    LogentriesError
-} from './error';
-import RingBuffer from './ringbuffer';
-import BunyanStream from './bunyanstream';
+const net = require('net');
+const reconnectCore = require('reconnect-core');
+const tls = require('tls');
+const urlUtil = require('url');
+const _ = require('lodash');
+const {Writable} = require('stream');
+
+const BadOptionsError = require('./optionsError');
+const build = require('./serialize');
+const defaults = require('./defaults');
+const getSafeProp = require('./loggerUtils');
+const InsightError = require('./error');
+const levelUtil = require('./levels');
+const text = require('./text');
+const {RingBuffer} = require('./ringBuffer');
 
 // patterns
 const newline = /\n/g;
 const tokenPattern = /[a-f\d]{8}-([a-f\d]{4}-){3}[a-f\d]{12}/;
 
 // exposed Logger events
-const errorEvent = 'error';
-const logEvent = 'log';
+const bufferDrainEvent = 'buffer drain';
 const connectedEvent = 'connected';
 const disconnectedEvent = 'disconnected';
+const errorEvent = 'error';
+const logEvent = 'log';
 const timeoutEvent = 'timed out';
-const drainWritableEvent = 'drain';
```

Comment:
Why are all these events being removed?

Reply:
Not used anywhere in the library, not sure why linter doesn't check for this 🤔